Library/test-helpers/lib.sh

```diff
@@ -1889,10 +1889,14 @@ true <<'=cut'
 
 Create container network.
 
-    limeconCreateNetwork NAME SUBNET
+    limeconCreateNetwork --ipv6 NAME SUBNET
```

Comment:
Just a nitpick, it is common to list optional options and arguments as `[--ipv6]`.